solution/backend/resources/admin.py

```diff
@@ -603,6 +628,25 @@ def get_queryset(self, request):
             Prefetch("group", FederalRegisterDocumentGroup.objects.all()),
         )
 
+    def get_content(self, obj):
+        if obj.id:
+            index = ContentIndex.objects.get(fr_doc=obj)
+            link = reverse("call-extractor", kwargs={"content_id": index.uid})
+            html = '<a class="button" href="{}">Get content</a>'.format(link)
+        else:
+            return "N/A"
+        return format_html(html)
```

Comment:
since else returns "N/A" does that mean only if obj.id we get format_html(html) so does it make sense to have it in that if statement?

Reply:
We dont need to have it format html in this case since its just a string

Reply:
corrected